.gitignore

```diff
@@ -32,7 +31,3 @@ local.properties
 build
 
 # Godot metadata files
-*.import
```

Comment:
Moved these into an ignore file in the demo folder itself, I am probably going to spend some more time moving all the ignore filters specific to a Godot project there. Makes it easier to use the demo stand alone, thinking about adding a second downloadable asset with a completely working demo project.